docs/Developers/Writing_Native_Bears.rst

```diff
@@ -110,9 +112,6 @@ You should now see an output like this on your command line:
     tutorial, else you will see a bunch of other outputs from other bears
     as well.
 
-The Bear class also supports ``warn`` and ``err`` to create ``WARNING`` and
-``ERROR`` messages respectively.
-
```

Comment:
Here you can describe ``logging`` support for ``WARNING`` and ``ERROR``.

Reply:
@Makman2 but explaining `logging` in writing bear doesnt make sense.

Reply:
Yes, at the beginning I wanted to put `The logging module also supports ``warning`` and ``error`` to create ``WARNING`` and ``ERROR`` messages respectively.` but someone told me that it wouldn't make sense to mention that..
I guess I should stick to what I did at the beginning after all?

Reply:
Hmm... maybe we should at least refer to the official documentation of `logging`?